sdk/containerregistry/ci.mgmt.yml

```diff
@@ -7,7 +7,7 @@ pr: none
 extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
-    SDKType: client
+    SDKType: mgmtclient
```

Comment:
need to add the type in the `service.projects` file
  <ItemGroup Condition="'$(SDKType)' == 'mgmtclient'">
    <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Azure.ResourceManager.*\**\*.csproj" />
  </ItemGroup>